x-pack/legacy/plugins/siem/public/components/charts/common.test.tsx

```diff
@@ -130,3 +105,36 @@ describe('getChartWidth', () => {
     expect(height).toEqual(defaultChartHeight);
   });
 });
+
+describe('checkIfAllValuesAreZero', () => {
+  const mockInvalidDataSets = [
+    [{ key: 'mockKey', color: 'mockColor', value: [{ x: 1, y: 0 }, { x: 1, y: 1 }] }],
+    [
+      { key: 'mockKeyA', color: 'mockColor', value: [{ x: 1, y: 0 }, { x: 1, y: 1 }] },
+      { key: 'mockKeyB', color: 'mockColor', value: [{ x: 1, y: 0 }, { x: 1, y: 0 }] },
+    ],
+  ];
+  const mockValidDataSets = [[{ key: 'mockKey', color: 'mockColor', value: [{ x: 1, y: 0 }] }]];
+
+  describe.each(mockInvalidDataSets)('return false', data => {
+    let result: boolean;
+    beforeAll(() => {
+      result = checkIfAllValuesAreZero(data);
+    });
+
+    it(`with data ${JSON.stringify(data)}`, () => {
+      expect(result).toBeFalsy();
+    });
+  });
+
+  describe.each(mockValidDataSets)('return true', data => {
+    let result: boolean;
+    beforeAll(() => {
+      result = checkIfAllValuesAreZero(data);
+    });
+
+    it(`with data ${JSON.stringify(data)}`, () => {
+      expect(result).toBeFalsy();
```

Comment:
the describe reads `return true`?? is this supposed to be a data set with all `0`s?

Reply:
Thank you, nice catch, I think I messed up the data format here again... https://jestjs.io/docs/en/api#describeeachtable-name-fn-timeout